webpack.mix.js

```diff
@@ -1,7 +1,7 @@
 let mix = require('laravel-mix');
 const webpack = require('webpack');
 
-mix.options({publicPath: 'public/'});
+mix.options({ publicPath: 'public/', processCssUrls: false });
```

Comment:
added this so the build step would not publish leaflet image files into the public directory